src/thunderbots/software/test/util/time/time.cpp

```diff
@@ -0,0 +1,118 @@
+#include "util/time/time.h"
+
+#include <gtest/gtest.h>
+
+#include <stdexcept>
+
+TEST(TimeTest, create_time_from_positive_seconds)
+{
+    Time time = Time::fromSeconds(1.0);
+    EXPECT_DOUBLE_EQ(time.getSeconds(), 1.0);
+}
+
+TEST(TimeTest, create_time_from_negative_seconds)
+{
+    Time time = Time::fromSeconds(-0.2);
+    EXPECT_DOUBLE_EQ(time.getSeconds(), -0.2);
+}
+
+TEST(TimeTest, create_time_from_zero_milliseconds)
+{
+    Time time = Time::fromMilliseconds(0);
+    EXPECT_DOUBLE_EQ(time.getMilliseconds(), 0);
+}
+
+TEST(TimeTest, create_time_from_negative_milliseconds)
```

Comment:
Missing test case for `create_time_from_positive_milliseconds`?

Reply:
Holding off on this until we decide what do do about the `Time` class

Reply:
Added